frontends/parsers/p4/p4parser.ypp

```diff
@@ -913,10 +921,10 @@ tableProperty
         { auto l = new IR::EntriesList(@3, *$6);
           auto id = IR::ID(@3+@7, "entries");
           $$ = new IR::Property(@3, id, $1, l, $2.isConst); }
-    | optAnnotations optCONST IDENTIFIER "=" initializer ";"
+    | optAnnotations optCONST nonTableKwName "=" initializer ";"
         { auto v = new IR::ExpressionValue(@5, $5);
-          auto id = IR::ID(@3, $3);
-          $$ = new IR::Property(@3, id, $1, v, $2.isConst); }
+          auto id = *$3;
+          $$ = new IR::Property(id, $1, v, $2.isConst); }
```

Comment:
You should still have the srcInfo for the property to give good error messages.
Use `$3->srcInfo`.